features/logging/cluster_log_forwarder.feature

```diff
@@ -421,3 +421,132 @@ Feature: cluster log forwarder features
+    And the expression should be true> cb.app_log_count_1 > 0
+
+    # check journal logs
+    When I execute on the pod:
+      | curl | -sk | -XGET | http://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"exists": {"field": "systemd"}}} |
+    Then the step should succeed
+    And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :journal_log_count_1 clipboard
+    And the expression should be true> cb.journal_log_count_1 > 0
+
+    # check logs in openshift* namespace
+    When I execute on the pod:
+      | curl | -sk | -XGET | http://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"regexp": {"kubernetes.namespace_name": "openshift@"}}} |
+    Then the step should succeed
+    And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :openshift_log_count_1 clipboard
+    And the expression should be true> cb.openshift_log_count_1 > 0
+
+    # check audit logs
+    When I execute on the pod:
+      | curl | -sk | -XGET | http://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"exists": {"field": "auditID"}}} |
+    Then the step should succeed
+    And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :audit_log_count_1 clipboard
+    And the expression should be true> cb.audit_log_count_1 > 0
+    """
+    # delete fluentd server
+    Given I ensure "fluentdserver" config_map is deleted from the "openshift-logging" project
+    And I ensure "fluentdserver" deployment is deleted from the "openshift-logging" project
+    And I ensure "fluentdserver" service is deleted from the "openshift-logging" project
+    # create another project to generate logs
+    Given I switch to the first user
+    Given I create a project with non-leading digit name
+    And evaluation of `project` is stored in the :proj1 clipboard
+    And I obtain test data file "logging/loggen/container_json_log_template.json"
+    When I run the :new_app client command with:
+      | file | container_json_log_template.json |
+    Then the step should succeed
+    # Again check logs in elasticsearch server
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-logging" project
+    And a pod becomes ready with labels:
+      | app=elasticsearch-server |
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    # check app logs
+    When I execute on the pod:
+      | curl | -sk | -XGET | http://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"match": {"kubernetes.namespace_name": "<%= cb.proj1.name %>"}}} |
+    Then the step should succeed
+    And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :app_log_count_2 clipboard
+    And the expression should be true> cb.app_log_count_2 > 0
+
+    # check journal logs
```

Comment:
The doc count may be changed before before the fluent server is deleted.  So the follow verify steps are invalid

Reply:
@anpingli Removed unnecessary checks, keeping only 'new app' log check.

Reply:
@gkarager  Could you attach the runner-v3 log?

Reply:
@anpingli Here is the link to pass log on runner-v3: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/318452/console